Yoast/javascript
# Add Polish word form functionality

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds word form functionality for Polish.
* [Yoast SEO Premium] Adds word form functionality for Polish.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Make sure all of the following forms of Polish words are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: _pies, psy, psa_
* Make sure all of the following forms of Polish words are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: _bezpieczna, niebezpiecznych_

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-554
